tests/src/kakarot/instructions/test_system_operations.py

```diff
@@ -87,7 +87,7 @@ async def test_call__should_transfer_value(self, system_operations, mint):
             system_operations.contract_address
         )
 
-    @pytest.mark.parametrize("salt", [0, 127, 256, 2**55 - 1])
+    @pytest.mark.parametrize("salt", [0])
```

Comment:
this will create a conflict with harsh, please don't update this test

Reply:
yessir